src/test/java/com/epam/tc/hw3/pageobjects/elements/BaseElements.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.tc.hw3.pageobjects.elements;
+
+import org.openqa.selenium.NoSuchSessionException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class BaseElements {
+
+    WebDriver webDriver;
+
+    public BaseElements(WebDriver webDriver) {
+        PageFactory.initElements(webDriver, this);
+        this.webDriver = webDriver;
+    }
+
+    public String openURL(String siteURL) {
+        webDriver.navigate().to(siteURL);
+        String actualURL = webDriver.getCurrentUrl();
+        return actualURL;
+    }
+
+    public String getTitle() {
+        String title = webDriver.getTitle();
+        return title;
+    }
+
+    public String getCurrentUrl() {
+        return webDriver.getCurrentUrl();
+    }
+
+    public boolean closeBrowser() {
+        webDriver.close();
+        try {
+            webDriver.getTitle();
+        } catch (NoSuchSessionException e) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
Why do you need method which close `WebDriver` inside `BaseElements` class

Reply:
It's from wh2.  Excercise 1 step 12 and Excercise 1 step 10 (steps where we check that browser is closed).
I posted for check hw2, and there i removed this steps (but still don't know correct it or not)

Reply:
Fixed